woocommerce/woocommerce-android
# [Simplified login] Align both variants in the unknown email screen

### Description
As discussed in p1667299605267799/1667290117.602879-slack-C046HDZL87J , we are changing the "unknown email" screen in control variant as well to show the "Create account" button.

### Testing instructions
1. Make sure you are using the control variant.
2. Open the app.
3. Type a non-existing WPCom account.
4. Confirm the screen show the "Create an account" button.
